.github/workflows/build-and-test-differential.yaml

```diff
@@ -105,12 +105,12 @@ jobs:
 
       - name: Run clang-tidy
         if: ${{ steps.get-changed-files.outputs.changed-files != '' }}
-        uses: autowarefoundation/autoware-github-actions/clang-tidy@v1
+        uses: yhisaki/autoware-github-actions/clang-tidy@enhance-clang-tidy
```

Comment:
- https://github.com/autowarefoundation/autoware-github-actions/pull/326 is merged.

Now we can make a release and point to it. @yhisaki do you need help with releasing?

Reply:
@xmfcx 
Thank you! I don't know much about a release and would like your help.

Reply:
Thank you very much for the clear instruction!!!
I released [v1.45.0](https://github.com/autowarefoundation/autoware-github-actions/releases/tag/v1.45.0) 🎉

Reply:
Perfect!